pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -216,8 +216,16 @@ func (r *{{ .Resource.Kind }}Reconciler) deploymentFor{{ .Resource.Kind }}({{ lo
 
 // labelsFor{{ .Resource.Kind }} returns the labels for selecting the resources
 // belonging to the given  {{ .Resource.Kind }} CR name.
+// Note that the labels follows the standards defined in: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
 func labelsFor{{ .Resource.Kind }}(name string) map[string]string {
-	return map[string]string{"type": "{{ lower .Resource.Kind }}", "{{ lower .Resource.Kind }}_cr": name}
+	return map[string]string{"app.kubernetes.io/name": "{{ lower .Resource.Kind }}",
```

Comment:
```suggestion
	return map[string]string{"app.kubernetes.io/name": "{{ .Resource.Kind }}",
```
I think we can leave is no lower case